### ISSUE TITLE
Support lazily represented content

### ISSUE THREAD

**Issue by panesofglass** [freya-fs/freya#127]
[Gitter chat October 19, 2015 9:11 AM](https://gitter.im/freya-fs/freya?at=5624fa072639469f2635a5d1)

We need something like Web API's `PushStreamContent` to allow for users to push large size content without having to buffer it into the Large Object Heap first. A good candidate for trying this is the Static File Server sample.

**Comment by kolektiv** [comment]
Yeah it would be good to have a chat about this one at some point - it's part of the OWIN spec I think (?) but I don't know whether I completely understand the expected usage model...
